.github/CODEOWNERS

```diff
@@ -146,6 +146,9 @@
 /sendmail/                               @dabcoder david.bouchare@datadoghq.com
 /signl4/                                 @rons4 ron@signl4.com @DataDog/ecosystems-review
 /sigsci/                                 @signalsciences info@signalsciences.com
+/shipyard/                               @shipyard support@shipyard.build 
+/shoreline/                              @ivang-shoreline @ciprian-shoreline @alinshoreline @cioc-shoreline @cristi-shoreline support@shoreline.io @DataDog/marketplace-review
```

Comment:
I think you may need to rebase from `master` since `@DataDog/marketplace-review` has changed their name to `@DataDog/ecosystems-review`.